lib/Graph.php

```diff
@@ -1612,7 +1612,7 @@ public function setType($resource, $type)
      * @param string|null $resource
      * @param string|null $lang
      *
-     * @return string A label for the resource.
+     * @return Literal|null An instance of Literal which contains the label or null.
```

Comment:
Other docstrings seem to use the fully qualified names of classes - should this be `\EasyRdf\Literal` instead of just `Literal`?

Reply:
I used the same approach as other function comments, e.g. in `Graph::newAndLoad`.

Reply:
But other functions use the full name, so we have an inconsistent naming approach. What is the safe way to do this?

Reply:
You are right. Both ways are used (and seem to work, probably since there is a namespace statement at the top of each file) and there is no apparent consistency. But I think that is a different problem, and should not hold back this PR. So I will approve it.

Reply:
OK